docs/sources/shared/datasources/tempo-editor-traceql.md

```diff
@@ -84,17 +91,30 @@ To create a query using autocomplete, follow these steps:
 
 ## View query results
 
-Query results for both the editor and the builder are returned in a table. Selecting the Trace ID or Span ID provides more detailed information.
+Query results for both the editor and the builder return in a table.
+You can interact with the data listed to view more details.
+
+Selecting the trace ID from the returned results opens a trace diagram.
+Selecting a span from the returned results opens a trace diagram and reveals the relevant span in the trace diagram.
+
```

Comment:
I know it's a lot of work, but the image is so busy that it's hard to parse the things described in lines 100-104. I'd suggest putting the image on line 106 here and then formatting lines 100-104 as a bullet list, each with an accompanying much smaller detail image to show the specific thing you're talking about. Hopefully you'd just be able to take cuts from the larger image to do this.

Reply:
I've actually found some text that has a better explanation in the TraceQL doc in Tempo OSS. I'm goign to use that and the screenshot from it. I think it will be a lot clearer.

Reply:
I rewrote this section to point to the doc that explains span details instead of duplicating it here. I also added some links that point to more details for other concepts.